pkgs/development/tools/pnpm/fetch-deps/pnpm-config-hook.sh

```diff
@@ -24,7 +24,12 @@ pnpmConfigHook() {
 
     echo "Installing dependencies"
 
-    pnpm install --offline --frozen-lockfile --ignore-script
+    if [[ -n "$pnpmWorkspace" ]]; then
+        extraPnpmInstallFlags="--filter=$pnpmWorkspace"
+    fi
+    runHook prePnpmInstall
+
+    pnpm install --offline --frozen-lockfile --ignore-script "$extraPnpmInstallFlags"
```

Comment:
```suggestion
    pnpm install --offline --frozen-lockfile --ignore-script "$pnpmInstallFlags"
```

Reply:
Nice, this also requires to document this variable and to allow others to use it. I revised the first commit to include this change, and I tested and documented this interface.